src/virtualbase.cpp

```diff
@@ -0,0 +1,97 @@
+#include "virtualbase.h"
```

Comment:
Move includes from header to source file for includes that are only used in the source file